aws-dynamodb-table/main.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.34"
```

Comment:
Not sure what version we should use.

Reply:
Může být. Aktualizuju to průběžně, tak spíš je dobrý dávat tam poslední verzi abys nepoužil nějakou featuru, která ve starší verzi není.

Reply:
Oki, hodim tam ">= 5.75" (nejnovejsi je 5.75.1, ale my myslim ty patch verze nedefinujeme ne?)